ports/psoc6/modtime.c

```diff
@@ -4,7 +4,7 @@
  * The MIT License (MIT)
  *
  * Copyright (c) 2013-2021 Damien P. George
- * Copyright (c) 2023 Infineon Technologies
+ * Copyright (c) 2023-2024 Infineon Technologies AG
```

Comment:
Some of them are 2023-2024 while in other files it is 2022-2024. This is intended?

Reply:
Yes, I tried to match the year we started to work on some of those files, till nowadays.
Also if they weren´t modified this year and they already have the header, I did no touch that. 
But, I might have inaccurately date them earlier or later... :(

Reply:
ok got it